website/project/decorators.py

```diff
@@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+
+def dev_only(func):
```

Comment:
Dev only is a handy decorator. I was planning on using it for the API playground page as well.

> On Jul 9, 2015, at 4:14 PM, Andy Boughton <notifications@github.com> wrote:
> 
> In website/project/decorators.py:
> 
> > @@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
> >  
> >      # Return decorator
> >      return wrapper
> > +
> > +
> > +def dev_only(func):
> Restricting certain views to dev-only was done at the request of @brianjgeiger ; pinging him for followup.
> 
> —
> Reply to this email directly or view it on GitHub.
>